mantidimaging/eyes_tests/base_eyes.py

```diff
@@ -64,6 +64,7 @@ def setUpClass(cls) -> None:
         cls.eyes_manager.set_batch(APPLITOOLS_BATCH_ID)
 
     def setUp(self):
+        self.app = QApplication.instance() or QApplication([])
```

Comment:
This might duplicate what is happening in `@start_application`. Maybe it could be a class level type annotation, near line 59.
`app: QApplication`